src/shared/dissect-image.c

```diff
@@ -479,6 +479,15 @@ int dissect_image(
         assert(!verity || verity->root_hash || verity->root_hash_size == 0);
         assert(!((flags & DISSECT_IMAGE_GPT_ONLY) && (flags & DISSECT_IMAGE_NO_PARTITION_TABLE)));
 
+#if !defined(GPT_ROOT_NATIVE) && !defined(GPT_ROOT_SECONDARY)
```

Comment:
Ifdeffery is never nice - if there's a preference to ifdef the declarations with _unused_ let me know

Reply:
Yeah, I think `_unused_` is preferred. At least that's what Lennart used in the latest PR.

Reply:
Updated, ptal.